sdk/keyvault/azkeys/custom_client.go

```diff
@@ -20,7 +20,8 @@ type ClientOptions struct {
 	azcore.ClientOptions
 }
 
-// NewClient creates a client that accesses a Key Vault's keys.
+// NewClient creates a client that accesses a Key Vault's keys. You should validate that vaultURL
+// references a valid Key Vault or Managed HSM. See https://aka.ms/azsdk/blog/vault-uri for details.
```

Comment:
Also do this for your cryptoclient. IIRC, you have one, but remote-only, right?

Reply:
No, we have only service clients (`azkeys.Client` has crypto methods).